File: sources/person.cpp
#include <string>
#include <map>

using namespace std;

struct Person
{
  string name = "Ethan Lerner";
  string aliases[1] = {"miapolis"};
  map<string, string> socials = {
      {"Email", "ethan@lerners.io"},
      {"GitHub", "https://github.com/miapolis"},
      {"Discord", "miapolis#????"},
      {"Spotify", "https://lerners.io/spotify"},
      {"Steam", "https://lerners.io/steam"}};
};

int main()
{
  struct Person ethan;
  return 0;
}

File: sources/person.ex
defmodule Person do
  defstruct name: "Ethan Lerner",
            aliases: ["miapolis"],
            socials: %{
              email: "ethan@lerners.io",
              github: "github.com/miapolis",
              # Send me a friend request!
              discord: "miapolis#????",
              spotify: "lerners.io/spotify",
              steam: "lerners.io/steam"
            }
end
